sklearn/metrics/pairwise.py

```diff
@@ -935,13 +939,46 @@ def _parallel_pairwise(X, Y, func, n_jobs, **kwds):
     if Y is None:
         Y = X
 
+    if n_jobs == 1:
```

Comment:
Only because a metric may not be picklable if n_jobs=1, and `delayed` obviously can't check that. I guess I should add a note.